### PR TITLE
Initial suggestions for README file inside solutions folder.

### LINKED ISSUE #56: Working on README file inside solutions folder for Calculator project.
Description:

We need to create a README.md for the Simple Basic Calculator project. The file should include:

Features: List the calculator's functionalities (e.g., add, subtract, factorial, etc.).
Contributors: Mention who worked on each feature.
How to Use: Provide instructions to clone the repo and run the script.

### PR DESCRIPTION
**Pull request by abeddost**
---
name: solution review 
about: A template PR for code review with a checklist
---

# Behavior

## Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [ ] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [ ] The function has correct type annotations
- [ ] The function is not called in the function file

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] There are no commented lines of code
- [ ] There are no `print` statements anywhere
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

closes #56

### CODE REVIEW

**Comment by Yevheniia-Rudenko** [comment]
Hello @abeddost! Thank you so much for creating the README! I reviewed it and pushed some additional changes. I found a bit more information from our peers and their functions in our chat, and I fixed the CI check issue (it was missing a closing ``` after the code and a link to Git).

If anyone is reviewing this branch and finds missing information, please feel free to update it or improve it.

**Comment by Ridwanayinde** [comment]
@abeddost @abeddost I have reviewed the README file. It looks good enough. The content is in a good state.

**Comment by Simi-Solola** [comment]
Well explanatory README, great work 👍

**Review by Yevheniia-Rudenko — DISMISSED** [review]
Can be merged into the main branch

**Review by Awaab98 — DISMISSED** [review]

File: solutions/README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+   cd ET6-foundations-group-12
+   ```
+
+### Requirements
+
+- Python 3.7 or higher
+
+### License
+
+This project is licensed under the MIT License. See the LICENSE file for more details.
```
[the remark this answers is not in the document]
Comment (Simi-Solola) [reply solutions/README.md]: This has been done

File: solutions/README.md [hunk 2]
```diff
@@ -1 +1,75 @@
 # Solutions
```
[the remark this answers is not in the document]
Comment (Simi-Solola) [reply solutions/README.md]: Successfully replaced

**Review by Yevheniia-Rudenko — APPROVED** [review]
Ready to be merged into the main
